freetextresponse/mixins.py

```diff
@@ -0,0 +1,32 @@
+"""
+Mixins for the Free Text Response XBlock
+"""
+import datetime
+import pytz
+from django.conf import settings
+
+TIME_ZONE = pytz.timezone(getattr(settings, 'TIME_ZONE', pytz.utc.zone))
+
+
+class EnforceDueDates(object):  # pylint: disable=too-few-public-methods
+    """
+    xBlock Mixin to allow xblocks to check the due date
+    (taking the graceperiod into account) of the
+    subsection in which they are placed
+    """
+
+    # Default values pulled from platform to None for tests
```

Comment:
I think you misworded this, specifically `pulled from platform _to_ None for tests`